components/script/dom/htmlimageelement.rs

```diff
@@ -141,11 +196,31 @@ impl HTMLImageElement {
                     let runnable = wrapper.wrap_runnable(runnable);
                     let _ = script_chan.send(CommonScriptMsg::RunnableMsg(
                         UpdateReplacedElement, runnable));
-                });
+                });*/
```

Comment:
Am unsure as to whether this part should be present. Seems that it should be present to trigger the reflow for image rendering. But since the image cache isn't responsible for fetching the images anymore, am confused as to whether this is required now.